### ISSUE TITLE
Andorid Local Notifications not received in app killed state

### ISSUE THREAD

**Issue by ninjastrikehero** [universal-tools/UTNotificationsFeedback#123]
Android Local Notifications was only received in minimized state and not in killed state with Version 1.8.4 on Below Devices
1. Oppo 6 with OS 10.3
2. One Plus 3 with OS 9
3. Oppo F9 with OS 9

**Comment by yuriy-universal-ivanov** [comment]
Hi @ninjastrikehero ,

Is the issue reproduced in other devices too, or it's specific for these 3?
As some devices, mostly of cheap segment, has a configuration and a tool that automatically force-stops any non-whitelisted application. Popular messengers are whitelisted by default, other apps can be whitelisted manually by user. It's done to save power, as these devices typically have very low capacity batteries. Is it your case maybe? Unfortunately we don't have any of the devices you've mentioned to test at. If it's the case, there is nothing really you can do.
Fortunately, only a very small number of devices behave that way.

**Comment by ninjastrikehero** [comment]
@yuriy-universal-ivanov :- We tested the issues on these devices so far. But rest of the devices such as Nokia 6.1 plus with OS 10, it is working fine. Also for the lower version of the plugin i.e. 1.8.2 it is working as expected on all the devices we tested so far.

**Comment by yuriy-universal-ivanov** [comment]
@ninjastrikehero ,

Yep, it looks like your case then. I checked, and indeed, you have to whitelist apps in Oppo devices so any background service can be run (which is required for notifications when app is closed): see https://www.quora.com/How-do-you-add-apps-into-Whitelist-in-OPPO-F1s-phone.
